docs/concepts/what-is-a-namespace.md

```diff
@@ -45,31 +45,30 @@ On self-hosted Temporal Clusters, you can register your Namespaces in the follow
 - In your Cluster setup, create your Namespaces, including the default, in your setup script.
   For example:
 
-  - If deploying through Docker Compose or using the [auto-setup image](https://github.com/temporalio/docker-builds/blob/main/docker/auto-setup.sh) in a custom Docker Compose application, the Namespace "default" is created, through the auto-setup script.
+  - If deploying through Docker Compose or using the [auto-setup image](https://github.com/temporalio/docker-builds/blob/ed312c749b4b8cf5f65f04eafc54e425582f6b9a/docker/auto-setup.sh) in a custom Docker Compose application, the Namespace "default" is created, through the auto-setup script.
```

Comment:
switching from main to a commit here?

Reply:
I keep making this update but it won't show